embed.lua

```diff
@@ -100,9 +101,9 @@ local function do_embed(debug_info)
                 out:write("unsigned char const* "..symbol.." = "..symbol.."_;\n")
                 out:write("int "..symbol.."_len = sizeof("..symbol.."_);\n")
 
-                out:write("#endif // ARCHITECTURE == "..name.."\n")
+                out:write("#endif // ARCHITECTURE_IS("..name..")\n")
 
-                print("      x"..name.." : "..tostring(#bin_data).." bytes")
+                print("       "..name.." : "..tostring(#bin_data).." bytes")
```

Comment:
nitpick:  Delete the `x`, instead of changing it to a `space`.  I'll do it after merging the PR.